src/main/java/edu/greenblitz/pegasus/commands/multiSystem/InsertIntoShooter.java

```diff
@@ -0,0 +1,46 @@
+package edu.greenblitz.pegasus.commands.multiSystem;
+
+import edu.greenblitz.GBLib.src.main.java.edu.greenblitz.gblib.subsystems.shooter.Shooter;
+import edu.greenblitz.pegasus.RobotMap;
+import edu.greenblitz.pegasus.commands.funnel.RunFunnel;
+import edu.greenblitz.pegasus.commands.intake.roller.RunRoller;
+import edu.greenblitz.pegasus.utils.DigitalInputMap;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.ParallelDeadlineGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+
+public class InsertIntoShooter extends SequentialCommandGroup {
+
+	private double startTime;
+	private boolean reported = false;
+
+	// AKA InsertoShooter @tal935
+	public InsertIntoShooter() {
+		addCommands(
+				new MoveBallUntilClick(),
+
+				//waits until the shooter is ready
```

Comment:
Then make a command that says wait until the shooter is ready. The command could be used in several places. Also a commandBuilder would be easier for this purpose